mifospay/src/main/AndroidManifest.xml

```diff
@@ -16,16 +16,11 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".merchants.ui.MerchantTransferActivity"></activity>
+        <activity android:name=".merchants.ui.MerchantTransferActivity" />
         <activity
             android:name=".auth.ui.LoginActivity"
-            android:theme="@style/AppTheme"
+            android:theme="@style/SplashTheme"
             android:windowSoftInputMode="stateVisible">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
-                <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
```

Comment:
I don't think it is wise to remove the intent filter, launcher applications use this intent-filter to start the app. Are you sure the app is working fine without the intent filter?